monkey/agent_plugins/credentials_collectors/chrome/src/chrome_browser_local_data.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path, PurePath
+from typing import Collection, Iterator, Optional
+
+from .chrome_browser_local_state import ChromeBrowserLocalState
+
+
+class ChromeBrowserLocalData:
+    """
+    The local data for a Chrome-based browser
+
+    :param local_data_directory_path: Path to the browser's local data directory
+    """
+
+    def __init__(self, local_data_directory_path: PurePath):
+        self._local_data_directory_path = local_data_directory_path
+        self._local_state = ChromeBrowserLocalState(local_data_directory_path / Path("Local State"))
```

Comment:
This may be problematic on Linux, where the master key is not stored in the local state. Perhaps the master key property should be abstract.